internal/contour/translator.go

```diff
@@ -238,7 +235,6 @@ func (t *Translator) addSecret(s *v1.Secret) {
 		return
 	}
 	t.Logger.Infof("caching secret %s/%s", s.Namespace, s.Name)
```

Comment:
delete this line, the caching is taken care of before this method is even called.